platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImpl.java

```diff
@@ -206,6 +210,25 @@ public Path getExportFile() {
     return contexts.values();
   }
 
+  private Properties getImportedCustomSystemProperties(Map<String, Object> metadata) {
```

Comment:
❗️ This method should be specific to getting older system properties such that for older version, you retrieve from the platform migratable and **expand** all properties which is currently missing here. For newer versions, it should simply load from the metadata the system properties that were previously exported.